arches/app/utils/index_database.py

```diff
@@ -251,7 +251,7 @@ def _index_resource_batch(resourceids):
 
```

Comment:
If we are not passing this to the process, can it be removed

Reply:
Refactored as discussed.